test/support/api.ex

```diff
@@ -0,0 +1,80 @@
+defmodule MongoosePushSupportAPI do
```

Comment:
```suggestion
defmodule MongoosePush.Support.API do
```
(dots in between name segments)